app/src/main/java/org/simple/clinic/search/results/PatientSearchResultsItemType.kt

```diff
@@ -48,105 +49,109 @@ sealed class PatientSearchResultsItemType : Item() {
       private val clock: Clock
   ) : PatientSearchResultsItemType() {
 
+    private lateinit var viewHolder: ViewHolder
+
+    private val genderImageView by unsafeLazy { viewHolder.patientsearchresult_item_gender }
```

Comment:
I personally don't see the benefit from doing this vs `lazy(LazyThreadSafetyMode.NONE)`, to be honest.

Reply:
Just so it's more readable.